src/components/ToDoItemEdit.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <div>
+    <div
+      class="relative sm:mx-auto mx-[16px] flex flex-col space-y-8 justify-center rounded-2xl bg-white border-black border-2 p-[18px] m-[50px] sm:w-[610px] w-[320px]"
+    >
+      <div class="flex w-full justify-between flex-row">
+        <div class="flex flex-col">
+          <input
+            class="w-full placeholder-black leading-[3rem] sm:text-5xl text-lg flex mt-4 font-bold"
+            type="text"
+            placeholder="Title"
+          />
+          <div class="flex w-full items-center sm:h-[12px] h-[5px] lg:hidden">
+            <img :src="dateIcon" class="lg:hidden mr-1 h-3 w-3" />
+            {{ localTodo.createdAt }}
+          </div>
+        </div>
+
+        <ToDoPriority v-model:priority="localTodo.priority" />
+        <BaseButton
+          @click="setPriority('Low')"
+          :color="'bg-green-500'"
+        ></BaseButton
+        ><BaseButton
+          @click="setPriority('medium')"
```

Comment:
Does this work as expected? You set `medium` as the priority but `Medium` is used in other files.. this is the reason why I said you have to use const instead of the string to avoid the typos. 
Please review the whole project and change the strings to the const.